ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ipv6-neighbors.j2

```diff
@@ -0,0 +1,32 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - ipv6 neighbors #}
+{% if ipv6_neighbor.cache_persistent is arista.avd.defined(true) or ipv6_neighbor.persistent.enabled is arista.avd.defined(true) or ipv6_neighbor.static_entries is arista.avd.defined %}
```

Comment:
as we have only these 3 keys inside `ipv6_neighbor` and we are checking them defined below, we can change this to -
`{% if ipv6_neighbor is arista.avd.defined %}`

Reply:
this is required to generate ! before any ipv6 neighbor command

Reply:
Okay, so it is required for the condition if `ipv6_neighbor.cache_persisten` or `ipv6_neighbor.persistent.enabled` is `false`.

We need to keep it then.